src/javascript/app/pages/user/new_account/virtual_acc_opening.js

```diff
@@ -121,7 +121,7 @@ const VirtualAccOpening = (() => {
                         loginid     : new_account.client_id,
                         token       : new_account.oauth_token,
                         is_virtual  : true,
-                        redirect_url: urlFor('new_account/welcome'),
+                        redirect_url: /gb/.test(residence) ? urlFor('new_account/realws') : urlFor('new_account/welcome'),
```

Comment:
do we want to check that client is also having status unwelcome instead of assuming?